data/deep missions.txt

```diff
@@ -806,29 +810,40 @@ mission "Deep: Singularity 0A"
 				`	"Yes. We used gravitational mapping equipment on Midgard and we found what looked like an artificial singularity."`
 					goto singularity
 				
-			`	The woman does not respond for a few seconds, then blurts out, "Don't lie to me, Captain. Otherwise I can make this experience very unenjoyable for you."`
+			apply
+				set "playing coy"
+			`	The woman does not respond for a few seconds. When she does begin speaking, you have to lean in to hear her. "Don't lie to me. I can make this experience very uncomfortable for you."`
 			choice
 				`	"Alright. We discovered an artificial singularity on Midgard."`
 			
 			label singularity
+			branch honest
+				not "playing coy"
```

Comment:
I'm unsure if this will work. I've attempted to try this in the past where a condition was set within a mission then in that same mission a branch existed that called upon the condition being set, but the branch would always act as if the condition was never set, even if it was.

Reply:
hm, you're probably right about that. I think player conditions are only set before and after the conversation. Probably worth an issue

Reply:
This makes me think that it will work

https://github.com/endless-sky/endless-sky/blob/a5a5cfec95eb2623167a8e703533aba45cf2fd12/source/ConversationPanel.cpp#L310-L320

Reply:
We'll see; it's been a while since I actually tested this method.

Reply:
It looks like that bit was last changed a year ago, but it was mostly reformatting and not something that likely changed how it was processing (change before that was 3 years ago)